src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -34,6 +34,12 @@
 #include "serviceRoutines/postUpdateContext.h"
```

Comment:
A line in CHANGE_NEXT_RELEASE about the issue fix should be added (have a look to the current CHANGES_NEXT_RELEASE file, it is plenty of examples :)